<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlSystemStateWidget</class>
 <widget class="QWidget" name="ControlSystemStateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;Bahnschrift&quot;;
color: rgb(0, 0, 0);
background-color: rgb(218, 170, 0);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QLabel" name="y_pos_label">
     <property name="text">
      <string>Y-Pos</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QLabel" name="pos_var_1">
     <property name="text">
      <string>0.000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="info_label">
     <property name="text">
      <string>Raw Telemetry</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2" colspan="4">
    <widget class="QLabel" name="temp_label">
     <property name="text">
      <string>Temperature</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="11" column="8" colspan="2">
    <widget class="QLabel" name="att_4">
     <property name="text">
      <string>0.000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="11" column="2" colspan="2">
    <widget class="QLabel" name="att_1">
     <property name="text">
      <string>0.000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="10">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="6">
    <widget class="QLabel" name="temp">
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="vel_var_label">
     <property name="text">
      <string>Vel-var</string>
     </property>
    </widget>
   </item>
   <item row="12" column="4" colspan="3">
    <widget class="QLabel" name="timestamp_label">
     <property name="text">
      <string>Timestamp</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QLabel" name="att_label">
     <property name="text">
      <string>Attitude (q)</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="z_pos_label">
     <property name="text">
      <string>Z-Pos</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="8" colspan="2">
    <widget class="QLabel" name="roll">
     <property name="text">
      <string>0.000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="5" colspan="2">
    <widget class="QLabel" name="pos_var_3">
     <property name="text">
      <string>0.000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLabel" name="y_pos">
     <property name="text">
      <string>0.000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="3" colspan="2">
    <widget class="QLabel" name="pos_var_2">
     <property name="text">
      <string>0.000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="x_pos_label">
     <property name="text">
      <string>X-pos</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="12" column="7" colspan="2">
    <widget class="QLabel" name="timestamp">
     <property name="text">
      <string>0:00:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="5" colspan="2">
    <widget class="QLabel" name="vel_var_3">
     <property name="text">
      <string>0.000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="batt_label">
     <property name="text">
      <string>Battery</string>
     </property>
    </widget>
   </item>
   <item row="8" column="7">
    <widget class="QLabel" name="pitch_label">
     <property name="text">
      <string>Pitch</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2" colspan="2">
    <widget class="QLabel" name="airspeed">
     <property name="text">
      <string>0.000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="7">
    <widget class="QLabel" name="roll_label">
     <property name="text">
      <string>Roll</string>
     </property>
    </widget>
   </item>
   <item row="11" column="6" colspan="2">
    <widget class="QLabel" name="att_3">
     <property name="text">
      <string>0.000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="2">
    <widget class="QLabel" name="vel_var_1">
     <property name="text">
      <string>0.000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="pos_var_label">
     <property name="text">
      <string>Pos-var</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QLabel" name="z_pos">
     <property name="text">
      <string>0.000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="7">
    <widget class="QLabel" name="yaw_label">
     <property name="text">
      <string>Yaw</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3" colspan="2">
    <widget class="QLabel" name="vel_var_2">
     <property name="text">
      <string>0.000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="2">
    <widget class="QLabel" name="airspeed_label">
     <property name="text">
      <string>Airspeed</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLabel" name="x_pos">
     <property name="text">
      <string>0.000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="batt">
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="8" colspan="2">
    <widget class="QLabel" name="yaw">
     <property name="text">
      <string>0.000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="8" colspan="2">
    <widget class="QLabel" name="pitch">
     <property name="text">
      <string>0.000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="11" column="4" colspan="2">
    <widget class="QLabel" name="att_2">
     <property name="text">
      <string>0.000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="8" colspan="2">
    <widget class="QLabel" name="z_acc">
     <property name="text">
      <string>0.000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="7">
    <widget class="QLabel" name="z_acc_label">
     <property name="text">
      <string>Z-Acc</string>
     </property>
    </widget>
   </item>
   <item row="5" column="8" colspan="2">
    <widget class="QLabel" name="y_acc">
     <property name="text">
      <string>0.000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="8" colspan="2">
    <widget class="QLabel" name="x_acc">
     <property name="text">
      <string>0.000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="7">
    <widget class="QLabel" name="y_acc_label">
     <property name="text">
      <string>Y-Acc</string>
     </property>
    </widget>
   </item>
   <item row="4" column="7">
    <widget class="QLabel" name="x_acc_label">
     <property name="text">
      <string>X-Acc</string>
     </property>
    </widget>
   </item>
   <item row="6" column="5" colspan="2">
    <widget class="QLabel" name="z_vel">
     <property name="text">
      <string>0.000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="5" colspan="2">
    <widget class="QLabel" name="y_vel">
     <property name="text">
      <string>0.000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="5" colspan="2">
    <widget class="QLabel" name="x_vel">
     <property name="text">
      <string>0.000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="2">
    <widget class="QLabel" name="x_vel_label">
     <property name="text">
      <string>X-Vel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="3" colspan="2">
    <widget class="QLabel" name="y_vel_label">
     <property name="text">
      <string>Y-Vel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="3" colspan="2">
    <widget class="QLabel" name="z_vel_label">
     <property name="text">
      <string>Z-Vel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
